coordinator/tests/services/translation/xml.spec.ts

```diff
@@ -1,8 +1,82 @@
-import {writeXmlStringCanonicalized} from "../../../src/services/serialisation/xml"
+import {writeXmlStringCanonicalized, writeXmlStringPretty} from "../../../src/services/serialisation/xml"
 import * as TestResources from "../../resources/test-resources"
 
 test("writeXmlStringCanonicalized returns correct value", () => {
   const actualOutput = writeXmlStringCanonicalized(TestResources.examplePrescription1.hl7V3SignatureFragments)
   const expectedOutput = TestResources.examplePrescription1.hl7V3FragmentsCanonicalized
   expect(actualOutput).toEqual(expectedOutput)
 })
+
+//TODO - add more tests to prove that XML is correctly canonicalized
```

Comment:
assume this is done?

Reply:
do we have tests around new `readXml` stuff?

Reply:
No, we've still only got the one test for canonicalization and it's not a very good test. Spotted this but didn't want to fix as part of this PR

Reply:
Will add the tests for the strip namespace stuff though, good spot

Reply:
Added tests.